flare/output.py

```diff
@@ -488,13 +494,20 @@ def write_gp_dft_comparison(
             stat += f' {mae_per_species[ele]:.2f}'
 
         # calculate potential and total energy
+        pot_en = 0
```

Comment:
Maybe tuck this into the if statement below?

Reply:
set the default to NaN in commit 485810945624f83687b50f0dc76d529dbc39eb34